apps/frontend/src/pages/quiz/quiz.tsx

```diff
@@ -26,6 +28,18 @@ const Quiz: React.FC = () => {
 					<Button isFluid label="CONTINUE" type="button" variant="dark" />
 				</div>
 			</div>
+			<img
+				alt="ripple-effect-bg"
```

Comment:
<img width="786" alt="image" src="https://github.com/user-attachments/assets/7655ec6c-440d-4b77-ab63-123dd4003a95">
Please, update the `alt` to the meaningful text or remove this attribute.